fitbenchmarking/utils/options.py

```diff
@@ -43,6 +47,11 @@ def __init__(self, file_name=None):
         self.use_errors = fitting.getboolean('use_errors')
 
         plotting = config['PLOTTING']
+        try:
+            self.make_plots = plotting.getboolean('make_plots')
+        except ValueError:
```

Comment:
Can you add a test that hits this exception?  That will keep the coverage percentage up

Reply:
Done